### PR TITLE
🌐 Add Chinese translations for `docs/zh/docs/advanced/settings.md`

### PR DESCRIPTION
**Pull request by ChuyuChoyeon**
For your approval

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 59a583640c40a73d3413b410328bc038576ed6d8 at: https://6484fb4918fa942d154ad80f--fastapi.netlify.app

**Comment by ChoyeonChern** [comment]
@tiangolo @Kludex please added labels

**Comment by ChoyeonChern** [comment]
@tiangolo  @Kludex

**Review by Chushine — APPROVED** [review]

**Review by Chushine — APPROVED** [review]

File: docs/zh/docs/advanced/settings.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+
+```Python hl_lines="3"
+import os
+
+name = os.getenv("MY_NAME", "World")
+print(f"Hello {name} from Python")
+```
+
+!!! tip
+    `os.getenv()` 的第二个参数是要返回的默认值。
```
Comment (lordqyxz) [inline docs/zh/docs/advanced/settings.md:64]:
```md
`os.getenv()` 
```
->
```  md
<a href="https://docs.python.org/3.8/library/os.html#os.getenv" class="external-link" target="_blank">`os.getenv()`</a>
```

Comment (ChoyeonChern) [reply docs/zh/docs/advanced/settings.md]: Thanks,The modification has been completed.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 323f328c521069a25fdd8ebd8fe7a3b006b262d2 at: https://648c4a21fca9190ceb2e84db--fastapi.netlify.app

**Review by lordqyxz — APPROVED** [review]

File: docs/zh/docs/advanced/settings.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+
+然后，您可以在应用程序中使用新的 `settings` 对象：
+
+```Python hl_lines="18-20"
+{!../../../docs_src/settings/tutorial001.py!}
+```
+
+### 运行服务器
+
+接下来，您将运行服务器，并将配置作为环境变
```
Comment (lordqyxz) [inline docs/zh/docs/advanced/settings.md:161]: 换行符多了

File: docs/zh/docs/advanced/settings.md [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+```
+
+在依赖项覆盖中，我们在创建新的 `Settings` 对象时为 `admin_email` 设置了一个新值，然后返回该新对象。
+
+然后，我们可以测试它是否被使用。
+
+## 从 `.env` 文件中读取设置
+
+如果您有许多可能经常更改的设置，可能在不同的环境中，将它们放在一个文件中，然后从该文件中读取它
+
```
Comment (lordqyxz) [inline docs/zh/docs/advanced/settings.md:286]: 换行符多了

File: docs/zh/docs/advanced/settings.md [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="1  11"
+    {!> ../../../docs_src/settings/app03_an/main.py!}
+    ```
+
+=== "Python 3.6+ 非注解版本"
+
+    !!! tip
+        如果可能，请尽量使用 `Annotated
```
Comment (lordqyxz) [inline docs/zh/docs/advanced/settings.md:361]: 换行符多了导致格式错误

Comment (ChoyeonChern) [reply docs/zh/docs/advanced/settings.md]: 已经修复

**Comment by tiangolo** [comment]
📝 Docs preview for commit 683845d96479fa416b177c722bac594e91a3eec6 at: https://648fc4e400fa3a10580d4e38--fastapi.netlify.app

**Pull request merged**